manager/controllers/utils/utils.go

```diff
@@ -171,7 +172,7 @@ func HasCondition(status *app.M4DApplicationStatus, cType app.ConditionType) boo
 func UpdateCondition(status *app.M4DApplicationStatus, cType app.ConditionType, reason string, message string) {
 	for ind, cond := range status.Conditions {
 		if cond.Type == cType && cond.Reason == reason {
-			if cond.Message != message {
+			if !strings.Contains(cond.Message, message) {
```

Comment:
Can you explain what is being done here

Reply:
Will add comments.
The idea is to aggregate as many errors as possible to report to the user at once, while avoiding duplication.
The entire error management is a mess (for some errors, we need to allow retrials and not raise Failure immediately). Errors are not detailed enough. I'll open a separate issue for this.

Reply:
Okay, a new issue will do

Reply:
https://github.com/IBM/the-mesh-for-data/issues/67